main/cmds.go

```diff
@@ -173,9 +188,21 @@ func enable(ctx *log.Context, h HandlerEnvironment, seqNum int) (string, error)
 		ctx.Log("event", "enable failed")
 	}
 
-	ctx.Log("event", "clearing setting files")
-	el := logging.New(nil)
-	settings.CleanUpSettings(el, h.HandlerEnvironment.ConfigFolder)
+	// clearing settings and scripts
```

Comment:
you could potentially isolate and put these lines (that are being repeated) in a dedicated method and call it from enablePre and enable methods, something like deleteSettingsAndScriptsForAllSequenceNumbersExceptThisOne(seqNum int)

Reply:
Added.